Dockerfile

```diff
@@ -0,0 +1,22 @@
+#get node image, comes with yarn preinstalled
+
+FROM node:10 As build
```

Comment:
I know that it is out of the scope of your work, but since node 10 is the last stable version and it works fine, could you update the other references to node images:
* https://github.com/FromDoppler/doppler-webapp/blob/master/build-w-docker.sh#L20
* https://github.com/FromDoppler/doppler-webapp/blob/master/watch-w-docker.sh#L19
* https://github.com/FromDoppler/doppler-webapp/blob/master/verify-w-docker.sh#L16
* Any other?

Reply:
will do